index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Mobile First Portfolio</title>
+  <link rel="stylesheet" href="style.css">
+  <style> @import url('https://fonts.googleapis.com/css2?family=Roboto:wght@100&display=swap'); </style>
+  <style> @import url('https://fonts.googleapis.com/css2?family=Roboto:wght@100&display=swap'); </style>
+  <style> @import url('https://fonts.googleapis.com/css2?family=Roboto:wght@100&display=swap'); </style>
+</head>
+<body>
```

Comment:
- [ ] Good work😃 on the project, but I see that the body of the project has an overflow scroll horizontally💯, so kindly try to fix this problem after that your project will be perfect😊

| Figma design | Your design| 
| --- | --- |
|   ![image](https://user-images.githubusercontent.com/83360636/212330925-757c292c-3564-43b0-974f-aef7dbdfd67e.png)|    ![image](https://user-images.githubusercontent.com/83360636/212330803-db1d476d-79f8-4aed-81b5-16f95544e62b.png) |